odoo-shippable/Dockerfile

```diff
@@ -1,145 +1,13 @@
 FROM vauxoo/odoo-80-image
-
 MAINTAINER Tulio Ruiz <tulio@vauxoo.com>
-COPY files/entrypoint_image /entrypoint_image
-COPY files/bash_colors /root/.bash_colors
-COPY files/etc_initd_postgresql /tmp/etc_initd_postgresql
-COPY files/get_odoo_git_compress.sh /get_odoo_git_compress.sh
 
 ENV RUN_COMMAND_MQT_10_ENTRYPOINT_IMAGE="/entrypoint_image" \
     INSTANCE_ALIVE="1" PSQL_VERSION="9.3" \
     REPO_REQUIREMENTS="/.repo_requirements" \
     WITHOUT_DEPENDENCIES="1" OCA_RUNBOT="1"
 
-# Create shippable user
-# Add git config data to root user
-RUN useradd -d /home/shippable -m -s /bin/bash -p shippablepwd shippable \
-    && echo 'shippable ALL=(ALL) NOPASSWD:ALL' | tee -a /etc/sudoers; mkdir -p /home/shippable/.ssh/; chown -R $USER:$USER /home/shippable/; chown -R shippable:shippable /home/shippable/ \
-    && git config --global user.name Shippable \
-    && git config --global user.email hello@shippable.com
-
-# Install postgres common files
-# Do not create postgres main clusters so we can name them mainVV where VV is postgres version 93 or 95 do far
-# Install basic postgres 9.3 and 9.5
-# Create the clusters with the proper naming
-# Set custom configuration of max connections, port and locks
-# Install pgbadger
-RUN apt-get update && apt-get install postgresql-common \
-    && sed -ri 's/#(create_main_cluster) .*$/\1 = false/' /etc/postgresql-common/createcluster.conf \
-    && apt-get install -y postgresql-9.3 postgresql-contrib-9.3 \
-    postgresql-9.5 postgresql-contrib-9.5 perl-modules make pgtune \
-    && mv /tmp/etc_initd_postgresql /etc/init.d/postgresql \
-    && chmod +x /etc/init.d/postgresql \
-    && rm -rf /var/lib/apt/lists/* \
-    && pg_createcluster 9.3 main -e=utf8 \
-    && pg_createcluster 9.5 main -e=utf8 \
```

Comment:
This was replaced with a different rationale explained at https://github.com/Vauxoo/docker-odoo-image/issues/117